android/cli/commands/_buildModule.js

```diff
@@ -561,6 +563,7 @@ AndroidModuleBuilder.prototype.generateModuleProject = async function generateMo
 	let buildGradleContent = await fs.readFile(path.join(this.moduleTemplateDir, 'build.gradle'));
 	buildGradleContent = ejs.render(buildGradleContent.toString(), {
 		compileSdkVersion: this.compileSdkVersion,
+		plugins: this.manifest.plugins.split(','),
```

Comment:
check if `this.manifest.plugins` is there before splitting

Reply:
Thanks!